src/main/java/core/basesyntax/CustomMathPow.java

```diff
@@ -1,14 +1,21 @@
 package core.basesyntax;
 
 public class CustomMathPow {
-
     /**
      * Написать метод pow(int number, int exponential), который является аналогом метода
      * Math.pow() и принимает 2 целых числа:
      *      number - число которое надо возводить в какую-то степень
      *      exponential - значение степени
      */
-    public int pow(int number, int exponential) {
-        return 0;
+    public int pow(int number, int exp) {
+        // negative numbers and negative exp remains unchecked
+        if (exp == 0) {
+            return 1;
+        }
+        int res = number;
+        for (int i = 1; i < exp; i++) {
+            res *= number;
+        }
+        return res < 0 ? Integer.MAX_VALUE : res;
```

Comment:
we shouldn't do this checking here

Reply:
This is a crutch, otherwise your tests will fall on value (number =9, exp =10), expected <Integer.MAX_VALUE>

Reply:
test was changed yesterday)